phpBB/phpbb/install/helper/navigation/install_navigation.php

```diff
@@ -58,15 +58,55 @@ public function get()
 						'stage'	=> true,
 						'order'	=> 1,
 					),
-					'obtain_data'	=> array(
-						'label'	=> 'STAGE_OBTAIN_DATA',
+					'obtain_admin_data'	=> array(
+						'label'	=> 'ADMIN_CONFIG',
 						'stage'	=> true,
+                        'is_link'	=> true,
+                        'link'	=> '/install/app.php/install',
```

Comment:
I don't think we should use hardcoded links in here. That's what we have the routing system for.